client/pages/index.js

```diff
@@ -27,36 +27,41 @@ export default function Home() {
    * @returns {Promise<void>}
    */
   async function loadNFTs() {
-    
     // For the Mumbai Testnet
     // const provider = new ethers.providers.JsonRpcProvider(rpc_url);
 
     // For the LocalHost
-    // const provider = new ethers.providers.JsonRpcProvider();
+    const provider = new ethers.providers.JsonRpcProvider();
```

Comment:
Please comment this line(**34**).

Reply:
@Rajdip019 I might be wrong, but you might have forget to comment it after the changes. line(33). Refer to https://github.com/Megabyte-143/certified_cliche/pull/33/files#r817398784